dot-net-sdk/validators/RuleValidator.cs

```diff
@@ -25,40 +26,73 @@ private static List<bool> EvaluateRuleCondition(SubjectAttributes subjectAttribu
 
     private static bool EvaluateCondition(SubjectAttributes subjectAttributes, Condition condition)
     {
-        if (subjectAttributes.ContainsKey(condition.attribute))
+        try
         {
-            subjectAttributes.TryGetValue(condition.attribute, out EppoValue outVal);
-            var value = outVal!;
-            Dictionary<OperatorType, Func<EppoValue, EppoValue, bool>> validationFunctions = new()
```

Comment:
I did not like this programming format of creating a bunch of lambdas in a dictionary

Reply:
Yeah the new way reads easier 📈  (and allows for the additional flow control for semver)